docs/operating-eck/troubleshooting/take-eck-dump.asciidoc

```diff
@@ -0,0 +1,125 @@
+:page_id: take-eck-dump
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Take ECK Dump
+
+
+Steps to take ECK dump:
+
+[float]
+== Prepare
+
+To download the ECK dump shell file, run:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+curl -L -O https://raw.githubusercontent.com/elastic/cloud-on-k8s/master/support/diagnostics/eck-dump.sh
+------------------------------------------------
+
+Change mode to executable.
+
+[source,bash]
+----
+chmod +x eck-dump.sh
+----
+
+[float]
+== Run
+
+Make sure to set the correct options. Check here for link:https://github.com/elastic/cloud-on-k8s/blob/master/support/diagnostics/eck-dump.sh[the latest version].
+
+[source,bash]
+----
+Options:
+  -N, --operator-namespaces     Namespace(s) in which operator(s) are running in (comma-separated list)
+  -n, --resources-namespaces    Namespace(s) in which resources are managed      (comma-separated list)
+  -o, --output-directory        Path to output dump files
+  -z, --create-zip              Create an archive with the dump files (implies --output-directory)
+  -v, --verbose                 Verbose mode
+----
+
+Command
+
+[source,bash]
+----
+./eck-dump.sh -o <path/to/output/dump/files> [-z] [-v] -N <operator namespace> -n <resources namespace>
+----
+
+
+[float]
+== Example
+
+Here is an example.
+
+- Assuming operator & resources are both in `elastic-system` namespace
+- Generate dump log to current folder with `eck-dump-log` prefix
+- Also zip it
+
+[source,bash]
+----
+./eck-dump.sh -o eck-dump-log -z -N "elastic-system" -n "elastic-system"
+----
+
+Sample output
+
+[source,bash]
+----
+ECK info dumped to eck-dump-log-15_ 7_2021_15_52_45.tgz
```

Comment:
Did you really get a white space?
```suggestion
ECK info dumped to eck-dump-log-15_07_2021_15_52_45.tgz
```

Reply:
Unfortunately, there's that space... I tested it again (I am pretty sure we can ignore the error message as I didn't add APM component when I did the test again.
```
./eck-dump.sh -o eck-dump-log -z -N "elastic-system" -n "elastic-system"
error: the server doesn't have a resource type "elasticmapsserver"
ECK info dumped to eck-dump-log-11_ 8_2021_19_41_02.tgz
```